package.json

```diff
@@ -20,6 +20,7 @@
     "@babel/preset-env": "^7.12.17",
     "@babel/preset-react": "^7.12.13",
     "@babel/preset-typescript": "^7.12.17",
+    "@types/jest": "^26.0.21",
```

Comment:
Does it matter than jest and types/jest are different versions? I know it matters with some things.

Reply:
It shouldn't as long as majors match (I just did latest of each)